lib/foreman_ansible/register.rb

```diff
@@ -137,6 +137,12 @@
                  :'api/v2/ansible_variables' => [:create]
                },
                :resource_type => 'AnsibleVariable'
+    permission :view_lookup_values,
```

Comment:
Permission `:view_ansible_roles` already got `{ :lookup_values => [:index] }`

Reply:
Since we don't even have permissions for the `lookup_values` resource, non-admin users can't access it (see https://github.com/theforeman/foreman/blob/develop/app/services/authorizer.rb#L34 and https://github.com/theforeman/foreman/blob/develop/app/graphql/mutations/base_mutation.rb#L31). 

You can refer to the output log below, which demonstrates the absence of filters related to the permission for the LookupValue class:
`no filters found for given permission`
`checking permission  for class LookupValue`
`organization_ids: []`
`location_ids: []`

Let me know if you need more details.